Dockerfile

```diff
@@ -1,28 +1,28 @@
-FROM daewok/sbcl:alpine AS build
-RUN apk update && apk upgrade
+FROM clfoundation/sbcl:2.1.5-alpine3.13 AS build
+RUN apk --no-cache add curl
 
 # Set working directory
 WORKDIR /opt/analyzer
-env HOME /opt/analyzer
+ENV HOME=/opt/analyzer
 
 # Pull down the latest Quicklisp
-ADD https://beta.quicklisp.org/quicklisp.lisp src/
+RUN mkdir build && curl https://beta.quicklisp.org/quicklisp.lisp -o build/quicklisp.lisp
 
-# install quicklisp
+# Install quicklisp
 COPY build/install-quicklisp.lisp build/
 RUN sbcl --script build/install-quicklisp.lisp
 
-# build the application
+# Build the application
 COPY build/build.lisp build/
 COPY src quicklisp/local-projects/analyzer
 RUN sbcl --script ./build/build.lisp
 
-## Build the runtime image
-FROM alpine
+# Build the runtime image
+FROM alpine:3.13
 WORKDIR /opt/analyzer
 
 # Copy over the analyzer code
-COPY --from=build /opt/analyzer/analyzer bin/
+COPY --from=build /opt/analyzer/bin/ bin/
```

Comment:
Does `sbcl --script ./build/build.lisp` produce a binary in the `/opt/analyzer/bin/` directory?

Reply:
Yes, this was a small change i snuck into this PR - it used to build `analyzer` at the top level - but i switched that to build it into a `bin` directory.

Reply:
👍